Gemfile

```diff
@@ -12,6 +12,12 @@ end
 
 gem 'listen', '< 3.1'
 
+if RUBY_VERSION.to_f < 2.2
```

Comment:
Not very critical, but this has strange behaviors when having versions like:
```ruby
"2.2.3".to_f < "2.10.1".to_f # => false ???
```
I generally prefer:

```
Gem::Version.new("2.2.3") < Gem::Version.new("2.10.1")
```